buildSrc/subprojects/parser/src/test/java/org/apache/calcite/test/DialectGenerateTest.java

```diff
@@ -177,4 +189,29 @@ private String readFile(Path path) {
   @Test public void processFileMultipleFunctionsSeparatedByLines() {
     assertFileProcessed("multiple_functions_separated");
   }
+
+  @Test public void processFileTokenAssignments() {
+    assertFileProcessed("token_assignments");
+  }
+
+  @Test public void processFileFunctionsAndAssignments() {
```

Comment:
nit: rename to processFileFunctionsAndTokenAssignments?

Reply:
Sure.